reference/dic_tags.rst

```diff
@@ -205,15 +205,15 @@ doctrine.event_listener
 **Purpose**: Add a Doctrine event listener
 
 For details on creating Doctrine event listeners, read the
-:doc:`/doctrine/event_listeners_subscribers` article.
+:doc:`Doctrine events </doctrine/events>` article.
```

Comment:
Do we actually need to put a title explicitly here ?

Reply:
Not strictly needed ... but useful because we may change the article title ... but we want the link text to always be "doctrine events".